prescient/data/simulation_state/time_interpolated_state.py

```diff
@@ -138,7 +138,8 @@ def get_current_actuals(self) -> Iterable[float]:
         but instead of returning arrays it returns a single value.
         '''
         if self._minutes_past_first_actuals == 0:
-            return self._inner_state.get_current_actuals()
+            for actual in self._inner_state.get_current_actuals():
+                yield actual

```

Comment:
Python has the `yield from` operator which could be used in-place of `return` to produce the same behavior. (Similar comment applied to lines 184-185 below.)

Reply:
Nice, I like it!  Told you I'm not a python developer...